docs/self-hosted/install/index.md

```diff
@@ -114,15 +115,19 @@ required packages:
         bin/mapit_make_css
         ./manage.py collectstatic
 
-* Create the database tables (do add an admin user when prompted, though you
-  can add one at any time with `./manage.py createsuperuser`):
```

Comment:
Not sure if this was a bug or intended behaviour (I don't get a prompt when running migrate), but I re-wrote this assuming migrate isn't supposed to add an admin.

Reply:
Migration used to prompt for creating a superuser, yeah, but that got removed from Django a long time ago.